specutils/spectra/spectrum1d.py

```diff
@@ -39,7 +40,8 @@ class Spectrum1D(OneDSpectrumMixin, NDDataRef):
         dimension of flux, or one greater than the last dimension of flux
         if specifying bin edges.
     wcs : `~astropy.wcs.WCS` or `~gwcs.wcs.WCS`
-        WCS information object.
+        WCS information object that either has a spectral component or is
```

Comment:
Just above here in `flux` I think you should specifically mention that `flux` can be an `NDCube` (and while you're at it, might as well mention it can be a `Spectrum1D`)